multiversion/src/main/scala/multiversion/loggers/ProgressBars.scala

```diff
@@ -1,8 +1,39 @@
 package multiversion.loggers
 
+import java.io.PrintWriter
+import java.time.Duration
+
+import moped.cli.Application
+import moped.progressbars.InteractiveProgressBar
 import moped.progressbars.ProgressBar
+import moped.progressbars.ProgressRenderer
 
 object ProgressBars {
+  def create(
+      app: Application,
+      renderer: ProgressRenderer,
+      intervalDuration: Duration = Duration.ofMillis(16)
+  ): ProgressBar = {
+    val out = new PrintWriter(app.env.standardError)
+
+    val ttyRenderer =
+      // Attempt to detect if we are writing to a TTY.
+      // See caveats at https://stackoverflow.com/q/1403772
+      if (System.console() != null) {
+        renderer,
```

Comment:
Interesting trailing comma 🤔  
https://scala-lang.org/files/archive/spec/2.13/01-lexical-syntax.html#trailing-commas-in-multi-line-expressions